espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/w3c/models/ActiveInputSources.java

```diff
@@ -12,11 +12,9 @@
  */
 public class ActiveInputSources {
     private Map<String, InputSource> inputSources = new HashMap<>();
+    private static ActiveInputSources globalActiveInputSources;
 
     public void addInputSource(InputSource inputSource) throws InvalidArgumentException {
```

Comment:
it does not throw now

Reply:
Yeah, that was on purpose. It validates in the process input algorithm instead.